cmd/install/install.go

```diff
@@ -303,5 +305,33 @@ func hyperShiftOperatorManifests(opts Options) ([]crclient.Object, error) {
 		return true
 	})...)
 
+	if opts.EnableAdminRBACGeneration {
+		// hypershift-client admin persona for hostedclusters and nodepools creation
+		clientClusterRole := assets.HyperShiftClientClusterRole{}.Build()
+		objects = append(objects, clientClusterRole)
+
+		clientServiceAccount := assets.HyperShiftClientServiceAccount{
+			Namespace: operatorNamespace,
+		}.Build()
+		objects = append(objects, clientServiceAccount)
+
+		clientRoleBinding := assets.HyperShiftClientClusterRoleBinding{
+			ClusterRole:    clientClusterRole,
+			ServiceAccount: clientServiceAccount,
+			GroupName:      "hypershift-client",
```

Comment:
I thought we were going to include these groups in the resources as well, no?

Reply:
maybe a misunderstanding. i had the impression that it was the general sentiment to keep non-k8s-native resources limited when possible. AppSRE can deal with group creation/maintenance, because OSD/ROSA dedicated-admin allows that.